lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -379,3 +395,17 @@ instance HasSeverityAnnotation TempDirLog where
     getSeverityAnnotation = \case
         MsgNoCleanup _ BracketStart -> Debug
         MsgNoCleanup _ _ -> Notice
+
+{-------------------------------------------------------------------------------
+                                    Mode
+-------------------------------------------------------------------------------}
+
+data Mode = Normal | Light (Maybe Blockfrost.TokenFile)
+  deriving (Show)
+
+modeFlag :: Parser Mode
+modeFlag = do
+    light <- flag False True $
+        mconcat [ long "light", help "Enable light mode" ]
```

Comment:
```suggestion
        mconcat [ long "light", help "Enable light-mode" ]
```
I think that it might be a good idea to stick to a single spelling. I have started to converge on "light-mode" with a hyphen.